src/core/url_store/atoms/urlStore.ts

```diff
@@ -36,10 +36,9 @@ export const urlStoreAtom = createAtom(
     { get, schedule, onAction, onInit, create, getUnlistedState },
     state: UrlData | null = null,
   ) => {
-    const isFeedSelectorEnabled = [
-      FeatureFlag.EVENTS_LIST__FEED_SELECTOR,
-      FeatureFlag.FEED_SELECTOR,
-    ].some((flag) => typeof configRepo.get().features[flag] !== 'undefined');
+    const isFeedSelectorEnabled =
+      configRepo.get().features['events_list__feed_selector'] ||
```

Comment:
Why use bare strings instead of constants (FeatureFlag.EVENTS_LIST__FEED_SELECTOR and FeatureFlag.FEED_SELECTOR) here?

Reply:
to be refactored during config simplification